.github/workflows/fluttertests.yml

```diff
@@ -0,0 +1,2 @@
+name: Dart/Flutter Package Analyzer
+uses: axel-op/dart-package-analyzer@v2.0.0
```

Comment:
This has to be wrapped by an `on:`  attribute from what the github workflow is telling me.